genesis/devnet.go

```diff
@@ -144,3 +146,114 @@ func NewDevnetWithConfig(config DevConfig) *Genesis {
 
 	return &Genesis{builder, id, "devnet"}
 }
+
+// NewDevnet create genesis for solo mode.
+func NewHayabusaDevnet() *Genesis {
+	launchTime := uint64(1526400000) // Default launch time 'Wed May 16 2018 00:00:00 GMT+0800 (CST)'
+	if launchTime == 0 {
+		launchTime = uint64(1526400000) // Default launch time 'Wed May 16 2018 00:00:00 GMT+0800 (CST)'
+	}
+
+	executor := DevAccounts()[0].Address
+	soloBlockSigner := DevAccounts()[0]
+	keyBaseGasPrice := thor.InitialBaseGasPrice
+
+	builder := new(Builder).
+		GasLimit(thor.InitialGasLimit).
+		Timestamp(launchTime).
+		ForkConfig(&thor.SoloFork).
```

Comment:
ForkConfig also used here, should be either passed in or returned, just to ensure the used forkconfig in the builder also used in other modules.